src/lsp/main.ts

```diff
@@ -64,6 +64,24 @@ function createClient(clojureLspPath: string): LanguageClient {
             "keep-require-at-start?": true,
         },
         middleware: {
+            didOpen: async (document, next) => {
+                if (document.uri.path.endsWith(config.REPL_FILE_EXT)) {
+                    return;
+                }
+                return next(document);
+            },
+            didSave: async (document, next) => {
+                if (document.uri.path.endsWith(config.REPL_FILE_EXT)) {
+                    return;
+                }
+                return next(document);
+            },
+            didChange: async (change, next) => {
+                if (change.document.uri.path.endsWith(config.REPL_FILE_EXT)) {
```

Comment:
We should probably use `isResultsDoc(doc)` from `results-doc.ts` here.

I noticed that there was prior art in this file doing it this way, so we should fix that one too.

A reason to check the extension is if we consider that Calva would have more than one output/REPL window, but if that happens we will need to revisit a lot of code, niot just this one.

Reply:
Done, I found no way to fix the one that was already being used on this file, we would need to convert a vscode.Uri to a vscode.TextDocument

Reply:
I see. Might be the reason it was not done like that. 😄